README.md

```diff
@@ -1,28 +1,37 @@
+# Knowledge Sharing Platform 🗒
 
-## ❓ Problem Statement
-Our Motive is to create a cross-platform application for student benefit. In the application student can share courses feedback, study notes, professor's feedback etc. So Before taking a courses other student can know more about professor and course. 
-Shared notes and important question can be useful guide for students to score good in that course. 
+Knowledge sharing platform application aims to get the students acquainted with the courses, professors, evaluation schemes. The said things can be achieved via gathering course feedback, professor feedback, and previous year grading schemes. This can be done by making a portal (just like a blog site) where people can share their honest reviews without being scrutinized much. Getting reviews from mass makes it easy for one to choose the particular course and get the right information about that course. Crowd-sourcing of materials and feedback would make it easy to organize resources at one place. It would be more reliable, save one’s time and avoid unnecessary bothering. Moreover, students can also get relevant course resources from the students who already opted for the course. This platform can also act as a general guidance portal.
 
+## Techstack 👩‍💻
 
-## :sparkles: Features
-- A Cross Platform Application so people can use them anywhere anytime.
-- Tech Stack : Django anb React/Angular(TBD)
+NextJS, TailwindCSS
 
+## Maintainers ☀️
 
-## Code Contributors
+| [@Ananya](https://github.com/Ananyaiitbhilai) | [@Anshuman Dhiman](https://github.com/AnshumanDhiman) |
+| --------------------------------------------- | ----------------------------------------------------- |
 
-The success of our vision to bring knowledge to every single hand depends on you. Even a small contribution helps. All forms of contributions are highly welcomed and valued.
+## Other details 📑
 
-## 🧑‍🏫 Maintainers 
+This project is in its secomd iteration. We might have NodeJS as backend. In future, might do clustering of students on the basis of the course taken.
 
-To keep the quality intact and drive the vision in the proper direction, we have maintainers.
+## Contribution Guidelines ✨
 
-> Maintainers are your friends forever. They are vastly different from moderators.
+Please go through the Wiki. The success of our vision to bring knowledge to every single hand depends on you. Even a small contribution helps. All forms of contributions are highly welcomed and valued.
 
-Currently, we have 2 active maintainers and we are expanding quickly.
-The task of maintainers is to review pull requests, suggest further quality additions and keep the work up to date with the current state of the world. <br/>
+-   [How to create a PR?](https://github.com/OpenLake/Knowledge-Sharing-Platform/wiki/How-to-create-a-PR%3F)
+-   [PR Format Guidelines](https://github.com/OpenLake/Knowledge-Sharing-Platform/wiki/PR-Format-Guidelines)
 
+## Set-up the project 💻
 
```

Comment:
can you update the details of setting up the project, so that it can give more clarity? Like how a user can get the .env credentials

Reply:
@AnshumanDhiman  I have updated the README.md. You can check.